spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -475,25 +475,25 @@ Make sure to add 'spring-boot-configuration-processor' as an optional dependency
 </dependency>
 ----
 
-The whitelisting support works only for uber-jar application artifacts! At the moment, the metadata properties are not retrievable from the dockerized application images directly — a dedicated companion metadata JAR is required.
+The allowlisting support works only for uber-jar application artifacts! At the moment, the metadata properties are not retrievable from the dockerized application images directly — a dedicated companion metadata JAR is required.
 
-The `configuration-properties.names` can whitelist properties that are defined in the property class annotated with `@ConfigurationProperties`. The properties defined with `@Value` annotations cannot be whitelisted.
+The `configuration-properties.names` can allowlist properties that are defined in the property class annotated with `@ConfigurationProperties`. The properties defined with `@Value` annotations cannot be allowlisted.
 ====
 
 
 [[spring-cloud-dataflow-stream-app-metadata-artifact]]
 ==== Creating and Using a Dedicated Metadata Artifact
 You can go a step further in the process of describing the main properties that your stream or task app supports by
 creating a metadata companion artifact. This jar file contains only the Spring boot JSON file about
-configuration properties metadata and the whitelisting file described in the previous section.
+configuration properties metadata and the allowlisting file described in the previous section.
 
 The following example shows the contents of such an artifact, for the canonical `log` sink:
 
 [source, bash]
 ----
 $ jar tvf log-sink-rabbit-1.2.1.BUILD-SNAPSHOT-metadata.jar
 373848 META-INF/spring-configuration-metadata.json
-   174 META-INF/spring-configuration-metadata-whitelist.properties
+   174 META-INF/spring-configuration-metadata-allowlist.properties
```

Comment:
We can't replace this property file name, until we move to the next full release of SCDF.

Reply:
No worries. Clearly, I was moving too quickly and got one I shouldn't have (I think I thought it was a sample rather than a file that really exists, but it doesn't matter - I shouldn't have changed that one).

Reply:
I fixed that, an extraneous comma in content that was new since my first commit, and the conflict.